### PR TITLE
Change to use hostNetwork for rest-server

### PR DESCRIPTION
**Pull request by Binyang2014**
To avoid CNI issue, change to use host network

### CODE REVIEW

File: src/rest-server/deploy/rest-server.yaml.template [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
       labels:
         app: rest-server
     spec:
-      hostNetwork: false
+      hostNetwork: true
```
Comment (abuccts) [inline src/rest-server/deploy/rest-server.yaml.template:32]: could you fix the cni issue instead? it's not a good practice to use `hostNetwork`

File: src/rest-server/deploy/rest-server.yaml.template [hunk 2]
```diff
@@ -144,7 +146,7 @@ spec:
 {% endif %}
         ports:
         - name: rest-server
-          containerPort: 8080
+          containerPort: 9186
```
Comment (abuccts) [inline src/rest-server/deploy/rest-server.yaml.template:149]: why specify `ports` if you use `hostNetwork`

File: src/rest-server/deploy/rest-server.yaml.template (hunk 1 above)
Comment (Binyang2014) [reply src/rest-server/deploy/rest-server.yaml.template:32]: CNI issue is not easy to fix and we don't have a solution yet. Waiting for @ydye investigation.

File: src/rest-server/deploy/rest-server.yaml.template (hunk 2 above)
Comment (Binyang2014) [reply src/rest-server/deploy/rest-server.yaml.template:149]: Will remove this

**Comment by Binyang2014** [comment]
> won't it affect other services? e.g. pylon

Currently, it only impact the service which need to access outside network. Seems other service works fine. I think we can put the PR here for deploy reference. Not hurry to merge.

**Comment by Binyang2014** [comment]
Close since we can remove CNI to fix this issue

**Pull request closed without merging**